app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] As demanded by project requirements, kindly ensure likes are also created for the current_user. The current user may not always be the user whose id is in the params.